### PR TITLE
feat: provide theming for email templates

### PR DESCRIPTION
**Pull request by amannn**
Adjusts the existing templates to match the Diakonie branding. The changes are rather minimal but we should test all email templates eventually.

### CODE REVIEW

File: templates/enquiry-notification-consultant.html [hunk 1]
```diff
@@ -48,15 +48,15 @@ <h1 style='color: #FFFFFF; font-size: 26px; font-weight: bold; line-height: 26px
                        <br/><br/>
                        Liebe Grüße
                        <br/>
-                       Ihr Diakonie Team
+                       Ihr Caritas Team
```
Comment (koepferd) [inline templates/enquiry-notification-consultant.html:51]: This should be `Diakonie`.

File: templates/message-notification-asker.html [hunk 2]
```diff
@@ -50,15 +50,15 @@ <h1 style='color: #FFFFFF; font-size: 26px; font-weight: bold; line-height: 26px
                        <br/>
                        <br/>
                        Liebe Grüße<br/>
-                       Ihr Diakonie Team<br/>
+                       Ihr Caritas Team<br/>
```
Comment (koepferd) [inline templates/message-notification-asker.html:53]: This should be `Diakonie`

**Review by koepferd — CHANGES_REQUESTED** [review]

**Comment by koepferd** [comment]
Open todos:

- [ ] await testing next week
- [ ] discuss integration of the templates (overwriting default files)

**Review by koepferd — APPROVED** [review]

File: templates/assign-enquiry-notification.html [hunk 3]
```diff
@@ -4,38 +4,38 @@
...
       <td width='5%' style="padding: 0;"><!-- spacer --></td>
       <td width='90%' style='background-color: #fff; padding: 0; '>
          <table style="table-layout: fixed; width: 100%" cellspacing='0' cellpadding='0' border="0">
-            <thead style='background-color: #CC1E1C;'>
+            <thead style='background-color: #5a2572;'>
                <td width="10%">
-                  <div style="margin: 16px; width: 100%">
-                     <a href=''><img style='width: 46px; height: 60px;' src='cid:company-logo.jpg'/></a>
+                  <div style="margin: 30px 16px 16px; width: 100%">
+                     <a href=''><img style='width: 40px; height: 40px;' src='cid:logo.png'/></a>
```
Comment (amannn) [inline templates/assign-enquiry-notification.html:17]: On Gmail Android the logo can overflow (same as with Caritas):

signal-2021-06-17-143340

**Comment by amannn** [comment]
Gmail web and GMX looks good to me.

Gmail:

Screenshot 2021-06-17 at 14 35 17


GMX:

Screenshot 2021-06-17 at 14 35 06

**Comment by koepferd** [comment]
Additionally to the logo overflow I have another issue on Android > Gmail App (Web Version is fine after accepting showing images) - only a part of the images are loaded:

photo_2021-06-17_14-55-44

Outlooks looks fine: Outlook Web on Android and Mac, Outlook App on Mac

**Comment by daho4b** [comment]
image

different distance between text and greeting

**Comment by daho4b** [comment]
image

no images are displayed (all-inkl webmail)

**Comment by koepferd** [comment]
I don't think the not showing images are general bugs. I checked the same mails again this morning and the images are loading - the client may load images depending on internet connection?

**Comment by amannn** [comment]
We discussed this PR in the daily today. The frontend part is finished and will be moved by @PepeCortisona to the right place. If I can help with something, please let me know!

**Pull request merged**